docs/network-policy.md

```diff
@@ -41,10 +41,54 @@ it, which determines its relative order among all tiers.
 **Note**: K8s NetworkPolicies will be enforced once all tiers have been
 enforced.
 
+### Tier CRDs
+
+Creating Tiers as CRDs allows users the flexibility to create and delete
+Tiers as per their preference i.e. not be bound to 5 static tiering options
+as was the case initially.
+
+An example Tier might look like this:
+```yaml
+apiVersion: security.antrea.tanzu.vmware.com/v1alpha1
+kind: Tier
+metadata:
+  name: mytier
+spec:
+    priority: 10
+    description: "my custom tier"
+```
+
+Tiers have the following characteristics:
+- Policies can associate themselves with an existing Tier by setting the `tier`
+  field in a Antrea NetworkPolicy CRD spec to the Tier's name.
+- A Tier must exist before an Antrea policy can reference it.
+- Policies associated with higher ordered (low `priority` value) Tiers are
+  enforced first.
+- No two Tiers can be created with overlapping priorities.
```

Comment:
nit: since for each Tier the priority is a number rather than a range, I don't feel `overlapping` makes sense here... how about `No two Tiers can be created with the same priority`?

Reply:
updated